app/docs/src/reference/audit-trail.md

```diff
@@ -0,0 +1,6 @@
+# Audit Trail Reference
+
+The Audit Trail feature gives you the ability to know what is going on inside
```

Comment:
We should add a sample of the structured log format IMO and show that metadata changes per type